manim/mobject/three_dimensions.py

```diff
@@ -165,15 +165,22 @@ def set_fill_by_checkerboard(self, *colors, opacity=None):
             face.set_fill(colors[c_index], opacity=opacity)
         return self
 
-    def set_fill_by_value(self, axes: "Mobject", colors: Union[Iterable[Color], Color]):
-        """Sets the color of each mobject of a parametric surface to a color relative to its z-value
+    def set_fill_by_value(
+        self,
+        axes: "Mobject",
+        colors: Union[Iterable[Color], Color],
+        axis: int = 2,
+    ):
+        """Sets the color of each mobject of a parametric surface to a color relative to its axis-value
 
         Parameters
         ----------
+        axis :
+            The chosen axis to use for the color mapping. (0 = x, 1 = y, 2 = z)
         axes :
-            The axes for the parametric surface, which will be used to map z-values to colors.
+            The axes for the parametric surface, which will be used to map axis-values to colors.
         colors :
-            A list of colors, ordered from lower z-values to higher z-values. If a list of tuples is passed
+            A list of colors, ordered from lower axis-values to higher axis-values. If a list of tuples is passed
             containing colors paired with numbers, then those numbers will be used as the pivots.
```

Comment:
The order of the parameters and the order of the docs are not in the right order, `axis` should be last.

Reply:
I will fix that. My bad.